app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self._table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self._table_name} (first_name, last_name) "
+            f"VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
+        cinema_cursor = self._connection.execute(
+            "SELECT *"
+            f"FROM {self._table_name}"
```

Comment:
Similar to the previous comment, ensure that the table name is not user-controlled to prevent SQL injection vulnerabilities.